web/src/actions/apd.js

```diff
@@ -56,29 +56,34 @@ export const updateApd = updates => dispatch => {
   }
 };
 
-export const selectApd = (id, { global = {}, pushRoute = push } = {}) => (
-  dispatch,
-  getState
-) => {
-  dispatch({ type: SELECT_APD, apd: getState().apd.byId[id] });
-  dispatch(updateBudget());
-  dispatch(pushRoute('/apd'));
+export const selectApd = (
+  id,
+  { deserialize = fromAPI, global = {}, pushRoute = push } = {}
```

Comment:
Incidentally, because `global` defaults to an empty object instead of defaulting to `window`, the `global.localStorage.setItem` on line 69 never does anything.  So that's why going straight to an APD on reload doesn't work right now.  😅

Reply:
Should there be an issue for fixing this?